src/scripts.js

```diff
@@ -43,18 +50,42 @@ const displayUserFriends = () => {
   user.friends.forEach((friend) => {
     foundFriend = userRepository.findUser(friend)
     friendsSection.insertAdjacentHTML('beforeend',
-    `<p>Name: ${foundFriend.name}</p>
-     <p>Email: ${foundFriend.email}</p>
-     <p>Saily Step Goal: ${foundFriend.dailyStepGoal}</p>
-     <p>Stride Length: ${foundFriend.strideLength}</p>`)
+    `<div class="friend">
+      <p>Name: ${foundFriend.name}</p>
```

Comment:
not sure how this change adds to the project.

Reply:
I didn't find anything about displaying friends' email. I took it out to make for cleaner appearance. Similarly, putting the paragraph text inside a div makes it easier for me to work in CSS.

Reply:
Makes it easier to work with in CSS